### ISSUE TITLE
Change Popup heading styles (integrate CC-vocabulary)

### ISSUE THREAD

**Issue by makkoncept** [creativecommons/ccsearch-browser-extension#37]
Change the HTML markup of the heading elements of the information-popup( popup which opens when you click on image thumbnail) of the extension and integrate with cc-vocabulary.

**How to do it**
Note - CC-vocabulary is already linked into the project, therefore you just need to add the appropriate classes and mimic the markup(if different) to change the styles.

Head over to the Heading section in [CC-Vocabulary docs](https://opensource.creativecommons.org/cc-vocabulary/styleguide/#/Elements/Heading). Inspect the heading element in dev-tools and copy its classes. Adding the same classes (after removing unnecessary current ones) to the button element in the extension will change the styles.

For eg - [This element](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/popup.html#L114) syntax should change as follows:

_Before_
```
<h3 class="attribution-label margin-bottom-small">Rich-text</h3>
```
_After_
```
<h6 class="vocab heading">Rich-text</h6>
```

**NOTE**:
- Change the tags of headings to h6 to be of the correct size.
- Remove `margin-bottom-small` class, as the heading already has some margin at the bottom.
- Also, Remove the outdated class definitions from CSS. Make sure you don't break anything, like the `.attribution-label` style definitions from [_popup.scss](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/sass/components/_popup.scss) will not be needed after this change(therefore, remove it) but removing `margin-bottom-small` class from [utilities.scss](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/sass/base/_utilities.scss) will surely break things.
- Only, add the styles to headings of information-popup (popup which opens after you click on image thumbnail).

**Comment by sharmaaditya570191** [comment]
Hi @makkoncept . Please have a look.

**Comment by makkoncept** [comment]
@sharmaaditya570191 please paste screenshots of the changes

**Comment by makkoncept** [comment]
fixed in #54
